test/SourceKit/Sema/availability_define.swift

```diff
@@ -0,0 +1,15 @@
+// RUN: %sourcekitd-test -req=sema %s -- %s \
+// RUN:  -Xfrontend -define-availability \
+// RUN:  -Xfrontend "_iOS8Aligned:macOS 10.10, iOS 8.0" | %FileCheck %s
+
+// REQUIRES: OS=macosx
+
+@available(_iOS8Aligned, *)
+public func onMacOS10_10() {}
+// CHECK-NOT: key.line: 7,
```

Comment:
Nitpick: can this be changed to `CHECK-NOT: key.line` or maybe `CHECK-NOT: key.description`?  Checking specific line numbers in a "-NOT" check would be really easy to accidentally break without us noticing the test was wrong.

Reply:
Good point, I've updated the test.